tests/e2e/scripts/hermes_bootstrap.sh

```diff
@@ -53,8 +53,8 @@ account_prefix = 'cosmos'
 key_name = 'rly01-gaia-a'
 store_prefix = 'ibc'
 max_gas = 6000000
-gas_price = { price = 0.00001, denom = 'uatom' }
-gas_multiplier = 1.2
+gas_price = { price = 0.005, denom = 'uatom' }
+gas_multiplier = 2
```

Comment:
TODO: revert this value to 1.2 (in all 4 places) once the gas estimation is fixed in the feemarket module and we upgrade to a new version

Reply:
We need to add the dynamic fees for hermes. I think that will fix the high `gas_multiplier`.

https://hermes.informal.systems/documentation/configuration/dynamic-gas-fees.html